### PR TITLE
Eng 2511 deprecate flowconfig in sdk

### PR DESCRIPTION
**Pull request by jpurusho65**
## Describe your changes and why you are making these changes
Remove the deprecated FlowConfig configuration parameter from publish_flow method
Add the missing K8sConfig for integrations config

## Related issue number (if any)
ENG 2511

## Checklist before requesting a review
- [ x] I have created a descriptive PR title. The PR title should complete the sentence "This PR...".
- [ x ] I have performed a self-review of my code.
- [ ] I have included a small demo of the changes. For the UI, this would be a screenshot or a Loom video.
- [ ] If this is a new feature, I have added unit tests and integration tests.
- [x ] I have run the integration tests locally and they are passing.
- [ ] I have run the linter script locally (See `python3 scripts/run_linters.py -h` for usage).
- [ ] All features on the UI continue to work correctly.
- [ ] Added one of the following CI labels:
    - `run_integration_test`: Runs integration tests
    - `skip_integration_test`: Skips integration tests (Should be used when changes are ONLY documentation/UI)

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -98,4 +98,4 @@ integration_tests/sdk/test-credentials.yml
 integration_tests/sdk/test-config.yml
 integration_tests/sdk/test-config-run.yml
 
-
+**/.devcontainer/
```
Comment (cw75) [inline .gitignore:101]: Typo?

Comment (jpurusho65) [reply .gitignore:101]: I added to exclude .devcontainer in .gitignore, as I normally develop in containers.  I'm not sure what team members use here, if they plan to use dev containers, we can standardize on that, it makes it easy. In getting my test environment to work natively on Mac, I had to manually install many python packages.  Please let me know if there is a concern in the approach.

Comment (cw75) [reply .gitignore:101]: Ah I see so it's not a unintentional commit. I'm fine with this then!

**Review by kenxu95 — COMMENTED** [review]
This looks good to me! Let me know when you get the tests working

**Comment by jpurusho65** [comment]
Inadvertently squashed UI commit along with this commit

**Pull request closed without merging**